### ISSUE TITLE
Integrating NppExec with Notepad++ macros

### ISSUE THREAD

**Issue by Gitoffthelawn** [d0vgan/nppexec#31]
Hi! Thank you for the powerful extension.

Is it possible to call a Notepad++ macro  from an NppExec script?

If not, what about the other way around, calling an NppExec script from a Notepad++ macro?

**Comment by d0vgan** [comment]
The menu Plugins ->NppExec -> Advanced Options allows to create menu items for selected NppExec's script.
The menu Macro -> Save Current Recorded Macro allows to save a macro as a menu item.
NppExec's command `npp_menucommand` allows to invoke any menu item - thus, once a desired macro has its associated menu item, it can be called by means of its menu item.
Finally, Settings -> Shortcut Mapper allows to assign shortcut keys to selected menu items, including those menu items that were associated with NppExec's script and/or macro.

**Comment by Gitoffthelawn** [comment]
I'm knowledgeable in the 1st, 2nd, and 4th steps, but not the 3rd step since I'm new to NppExec.  I think that 3rd step is likely the linchpin I need to make it all work together.  Thank you for providing it! 

BTW, thank you again for NppExec.  It's really quite impressive and powerful.  One little hiccup I found was that the syntax for creating a label didn't seem well documented (or at all?).  After a bunch of experimenting, I finally figured out the syntax to use so that my `if ... goto` commands would work as expected.  Might be good to add some examples somewhere.
